lib/rubygems/util/list.rb

```diff
@@ -2,6 +2,8 @@ module Gem
   List = Struct.new(:value, :tail)
 
   class List
+    include Enumerable
```

Comment:
If you include `Enumerable` you don't need the explicit `find` implementation anymore. The `to_a` implementation could go too if `Struct` didn't have a default one.